lib/factories/ppt/slides.js

```diff
@@ -279,10 +310,10 @@ class SlideFactory {
         let slideKey = `ppt/slides/${slide.name}.xml`;
         let chartKey = `ppt/charts/${chartName}.xml`;
 
-        let chartX = options.x ? PptxUnitHelper.fromPixels(options.x) : 0;
-        let chartY = options.y ? PptxUnitHelper.fromPixels(options.y) : 0;
-        let chartCx = options.cx ? PptxUnitHelper.fromPixels(options.cx) : PptxUnitHelper.fromPixels(600);
-        let chartCy = options.cy ? PptxUnitHelper.fromPixels(options.cy) : PptxUnitHelper.fromPixels(400);
+        let chartX = options.x !== undefined ? PptxUnitHelper.fromPixels(options.x) : 0;
```

Comment:
same comment about the default object assignment above.

Reply:
Done.